packages/renderer/src/lib/ingresses-routes/IngressesRoutesList.svelte

```diff
@@ -0,0 +1,143 @@
+<script lang="ts">
+import { onDestroy, onMount } from 'svelte';
+
+import type { Unsubscriber } from 'svelte/store';
+import { filtered, searchPattern } from '../../stores/deployments';
```

Comment:
Needs to come from ingress + routes store - currently it is filtering deployments so I don't see the empty screen, or see it while there are still ingresses. :)